servers/physics/body_pair_sw.cpp

```diff
@@ -71,6 +72,7 @@ void BodyPairSW::contact_added_callback(const Vector3 &p_point_A, const Vector3
 
 	contact.acc_normal_impulse = 0;
 	contact.acc_bias_impulse = 0;
+	contact.acc_bias_impulse_center_of_mass = 0;
```

Comment:
Accumulated bias impulse applied to the center of mass (that won't cause rotation)